internal/provider/namespace_resource_test.go

```diff
@@ -47,6 +48,17 @@ resource "temporal_namespace" "test" {
 					resource.TestCheckResourceAttr("temporal_namespace.test", "owner_email", "updated@example.org"),
 				),
 			},
+			// Namespace already exists Error
```

Comment:
Minor: we should ideally create a new test case for this scenario

Reply:
Moved to a new Test Case